tests/MockAPNSServer/server.js

```diff
@@ -0,0 +1,55 @@
+const http2 = require("http2");
+const fs = require("fs");
+const uuid = require("uuid");
+
+const server = http2.createSecureServer({
+  key: fs.readFileSync("test-key.pem"),
+  cert: fs.readFileSync("test-cert.pem"),
+});
+server.on("error", (err) => console.error(err));
+
+var requestNumber = 0;
+var sessions = [];
+
+server.on("session", (session) => {
+  sessions.push(session);
+});
+
+server.on("stream", (stream, headers) => {
+
+  // stream is a Duplex
```

Comment:
Nitpick

```suggestion
  // Stream is a duplex
```

Unless "Duplex" is an actual name, and not just an attribute of the stream.

Reply:
Good call – the fact that a stream is bi-directional is an implementation detail – this comment is left from the example I pulled it from, so it can just be deleted. And was in 869b57f30e9fe088e176e27ef5bf145b0e397de2